pkg/util/crd_defaults.go

```diff
@@ -96,46 +96,9 @@ func GetHubDefaultPersistentStorage() *blackduckv1.BlackduckSpec {
 		},
 		CertificateName: "default",
 		Type:            "Artifacts",
```

Comment:
Just checking to make sure you don't want a default ExposeService or Version here.

Reply:
I don't think we want to expose the service by default as it may not be secure. 
Regarding the blackduck version, I personally don't have a preference here. If we do specify a default version, then we will need to update it for each release of synopsysctl.  Also, which version should be the default? 
@msenmurugan  @rrati  ^^